### ISSUE TITLE
Not working in Linux

### ISSUE THREAD

**Issue by james-tate** [CANBus-Triple/CANBus-Triple#37]
I can not get this to work on my Linux machine. I have tried various Arduino versions and have followed the getting started page exactly. Out of the box I get this error: 

CANBusTriple.ino:11:20: fatal error: CANBus.h: No such file or directory

If I clear that up, I then get errors about variables being undefined.

Have you thought about making a build script so the crappy Arduino IDE is not needed?

**Comment by etx** [comment]
Hrmmmmmmm. 

How about trying the board definition by adding this URL: http://files.canb.us/arduino/package_canbustriple_index.json

Add it under Preferences > Additional Boards Manager URLs field.

Also it was last tested with Arduino 1.6. If new versions have come out they tend to make breaking changes in every release so I would give 1.6 a try. 

The new CBT runs on ARM and uses Javascript leaving Arduino in the dust. It's been in testing for a long while now, hope to finish it up soon.

**Comment by james-tate** [comment]
Nope, The Linux 1.6 version does not support the additional boards option. I wrote a makefile for compiling and a python script that forces reboot to bootloader for flashing.

Which ARM controller are you using for the new version? Also why use Javascript as opposed to a conventional embedded language like C/C++? Do you feel you could run into performance issues since JS uses more memory and can be slower?
